geopandas/io/sql.py

```diff
@@ -47,7 +47,7 @@ def read_postgis(sql, con, geom_col='geom', crs=None, hex_encoded=True,
         raise ValueError("Query missing geometry column '{}'".format(geom_col))
 
     def load_geom(x):
-        return shapely.wkb.loads(str(x), hex=hex_encoded)
+        return shapely.wkb.loads(x, hex=hex_encoded)
```

Comment:
The automatic conversion to `str` that I think may solve the WKBElement read issue turns out to break when trying sqlite access.

Reply:
What is the "the WKBElement read issue" ?

Reply:
Sorry, #546